src/uct/ib/base/ib_device.h

```diff
@@ -61,7 +61,8 @@ enum {
     UCT_IB_DEVICE_FLAG_DC_V2    = UCS_BIT(7),   /* Device supports DC ver 2 */
     UCT_IB_DEVICE_FLAG_AV       = UCS_BIT(8),   /* Device supports compact AV */
     UCT_IB_DEVICE_FLAG_DC       = UCT_IB_DEVICE_FLAG_DC_V1 |
-                                  UCT_IB_DEVICE_FLAG_DC_V2 /* Device supports DC */
+                                  UCT_IB_DEVICE_FLAG_DC_V2, /* Device supports DC */
+    UCT_IB_DEVICE_FLAG_DEVX     = UCS_BIT(9)
```

Comment:
doesn't seem like we really need this flag

Reply:
Device may have DEVX support or may not

Reply:
can we move this flag to uct_ib_mlx5_md?

Reply:
yes

Reply:
yes